libs/components/core/testing/src/media-query/media-query-testing.service.ts

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable @nx/enforce-module-boundaries */
+import {
+  ElementRef,
+  Injectable,
+  OnDestroy,
+  RendererFactory2,
+  inject,
+} from '@angular/core';
+import {
+  SKY_MEDIA_BREAKPOINT_DEFAULT,
+  SKY_MEDIA_BREAKPOINT_TYPES,
+  SKY_MEDIA_BREAKPOINT_TYPE_DEFAULT,
+  SkyMediaBreakpointType,
+  SkyMediaBreakpoints,
+  SkyMediaQueryListener,
+  SkyMediaQueryServiceInterface,
+  SkyResizeObserverMediaQueryService,
+  SkyResizeObserverMediaQueryServiceInterface,
+  toSkyMediaBreakpoints,
+} from '@skyux/core';
+
+import {
+  BehaviorSubject,
+  Observable,
+  Subscription,
+  firstValueFrom,
+} from 'rxjs';
+
+import { SkyMediaQueryTestingController } from './media-query-testing.controller';
+
+/**
+ * @internal
+ */
+@Injectable({ providedIn: 'root' })
```

Comment:
This shouldn't be provided in root, correct? Shouldn't it only be provided by `provideSkyMediaQueryTesting()`?

Reply:
Doh!